tailwind.config.ts

```
@@ -1,63 +1,64 @@
+          '5': 'hsl(var(--chart-5))'
+        }
+      },
+      borderRadius: {
+        lg: 'var(--radius)',
+        md: 'calc(var(--radius) - 2px)',
+        sm: 'calc(var(--radius) - 4px)'
+      }
+    }
   },
-  plugins: [require("tailwindcss-animate")],
+  plugins: [require('tailwindcss-animate')]
```

ESLint

[error] A `require()` style import is forbidden.

(@typescript-eslint/no-require-imports)